qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/services/codeaction/QuteCodeActionForSimilarTextSuggestionsTest.java

```diff
@@ -186,4 +187,66 @@ public void similarTextSuggestionQuickFixForUndefinedObjectUsingLoopSectionMetad
 						ConfigurationItemEditType.update, "ignore", //
 						d)));
 	}
+
+	@Test
+	public void similarTextSuggestionQuickFixForUndefinedMethod() throws Exception {
+		String template = "{@java.lang.String string}\r\n" + //
+				"{string.charA()}";
+
+		Diagnostic d = d(1, 8, 1, 13, //
+				QuteErrorCode.UnknownMethod, //
+				"`charA` cannot be resolved or is not a method of `java.lang.String` Java type.", //
+				DiagnosticSeverity.Error);
+		d.setData(new JavaBaseTypeOfPartData("java.lang.String"));
+
+		testDiagnosticsFor(template, d);
+		testCodeActionsFor(template, d, //
+				ca(d, te(1, 8, 1, 13, "charAt")));
+	}
+
+	@Test
+	public void similarTextSuggestionQuickFixForUndefinedMethodWithResolver() throws Exception {
+		String template = "{@java.lang.String string}\r\n" + //
+				"{string.ifTru()}";
```

Comment:
Please replace `ifTruthy` with `orEmpty` value resolver because `ifTruthy` is an internal value resolver.